dbus_next/_private/unmarshaller.py

```diff
@@ -66,6 +69,24 @@ def align(self, n):
             return b''
         return self.read(padding)
 
+    def read_endian(self, _=None):
+        unix_fds = array.array("i")
+
+        if self.sock:
+            try:
+                msg, ancdata, *_ = self.sock.recvmsg(1, socket.CMSG_LEN(16 * unix_fds.itemsize),
```

Comment:
I think we should replace `1` with the size of fixed message header. Also, why `16`? Let's at least extract magic number into a constant.

Reply:
Done